### DISCUSSION TITLE
I wanna do dynamic local texts

### DISCUSSION THREAD

**Discussion by k61b** [vercel/next.js#52345] [App Router]
We are using app router and we made our project like [this](https://github.com/vercel/next.js/tree/canary/examples/app-dir-i18n-routing). How can we add dynamic local texts and use?
Like:
```
"welcome_message": "Welcome, {{name}}!"
i18next.t('welcome_message', { name: 'Trabzon' })
Welcome, Trabzon!
```

**Comment by yusufakcekoce** [comment]
I'm having the same problem, I haven't found a solution yet
